### PR TITLE
Use node 16.x in tests and development

### PR DESCRIPTION
**Pull request by ajacksified**
This PR updates the .nvmrc to use Node 16, updates the CI / CD workflows to include 16 in the test matrix and for builds, and  rebuilds the package-lock.json using NPM 8.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,9 +19,7 @@ jobs:
       matrix:
         # Available OS's: https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
         os: [ubuntu-20.04, windows-2019, macos-10.15]
-        # node-version: [16.x, 14.x, 12.x]
-        # FIXME: find out why Node 16 crashes.
```
Comment (ThisIsMissEm) [inline .github/workflows/ci.yml:23]: This is still crashing: https://github.com/inrupt/solid-client-js/runs/4537142263?check_suite_focus=true

Comment (matthieubosquet) [reply .github/workflows/ci.yml:23]: It's an env variable problem, I'll fix it.

**Review by matthieubosquet — APPROVED** [review]
CI now passing. LGTM.

**Comment by matthieubosquet** [comment]
@ajacksified I just disabled the browser tests waiting for a fix. So I think we can merge this ;)

**Comment by ThisIsMissEm** [comment]
@ajacksified looks like snyk is failing or hung for some reason? maybe do a rebase and force-push?

**Pull request merged**